### PR TITLE
added rst file for the Release Notes 3.2.1

### PR DESCRIPTION
**Pull request by svekars**

### CODE REVIEW

File: pages/release-notes/v3-2-1-grizzly-full.rst [hunk 1]
```diff
@@ -0,0 +1,511 @@
+.. index:: Release Notes: Fuel 3.2
+
+.. _RelNotes_3.2.1:
+
+Release Notes for Mirantis OpenStack version 3.2.1
+================================================
```
Comment (e0ne) [inline pages/release-notes/v3-2-1-grizzly-full.rst:6]: Incorrect underline symbols count

File: pages/release-notes/v3-2-1-grizzly-full.rst [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+
+.. contents:: :local:
+  :depth: 1
+  :backlinks: none
+
+Overview
+---------
+
```
Comment (e0ne) [inline pages/release-notes/v3-2-1-grizzly-full.rst:14]: Incorrect underline symbols count

File: pages/release-notes/v3-2-1-grizzly-full.rst [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+This version is a maintenance release to Mirantis OpenStack 3.2 and contains defect fixes and several resolutions for issues initially reported against version 3.2.  You do not need to install version 3.2 prior to installing 3.2.1 as the 3.2.1 release is cumulative with version 3.2.
+
+Mirantis OpenStack is a complete solution that includes three major components:
+
+* Fuel lifecycle management application
+* Mirantis OpenStack hardened packages
```
Comment (e0ne) [inline pages/release-notes/v3-2-1-grizzly-full.rst:20]: Need to add blank line between list items

File: pages/release-notes/v3-2-1-grizzly-full.rst [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+* Mirantis Support
+
+These release notes supplement the product documentation and list enhancements, resolved issues and known issues in this version. 
+
+What is Mirantis OpenStack?
+------------------------------
```
Comment (e0ne) [inline pages/release-notes/v3-2-1-grizzly-full.rst:26]: Incorrect underline symbols count

**Comment by e0ne** [comment]
Please, check for underlined symbols for headers and blank lines between lists items. They leads to build warnings

**Comment by mihgen** [comment]
contains irrelevant release notes for 3.2.1

**Pull request closed without merging**